docs/config/README.md

```diff
@@ -57,8 +58,32 @@ This is the list of prompt-wide configuration options.
 
 # Disable the newline at the start of the prompt
 add_newline = false
+# Overwrite a default_prompt_order and  use custom prompt_order
+prompt_order=["rust","line_break","package","line_break","character"]
 ```
 
+### Default prompt order
+```default_prompt_order``` is used as ```prompt_order``` when config have no options named as ```prompt_order``` or when you have set ```prompt_order = []```
```

Comment:
A little bit of nitpicking around how this option is documented.

```suggestion
The ```default_prompt_order``` configuration option is used to define the order in which modules are shown in the prompt. If no ```default_prompt_order``` is provided, the default is as shown:
```